xradar/io/export/cfradial1.py

```diff
@@ -304,6 +306,21 @@ def to_cfradial1(dtree=None, filename=None, calibs=True):
 
     dataset.attrs = dtree.attrs
 
+    # Find variables with >= 2 dimensions
+    multidim_variables = [
+        var for var in dataset.data_vars if len(dataset[var].dims) >= 2
+    ]
+
+    # Set _FillValue attribute to -32768 for variables with >= 2 dimensions
+    fill_value = -32768
```

Comment:
Is this _FillValue documented somewhere? Is it in the CfRadial1 standard?

Reply:
Yeah, so I designed it according to the file DATASETS.fetch("cfrad.20080604_002217_000_SPOL_v36_SUR.nc"). And, I have also noticed that number -32768 in other CfRadial data before this. I think this is the most common fillvalue used in the CfRadial1 data. Did I answer your question?

Reply:
I've quickly swept the CfRadial docs and there is no default `_FillValue` mentioned.

I'd suggest to use the netcdf default `_FillValue` in these cases and probably make this parametrizable.

@mgrover1 Do you have a bit more insight here. Maybe ping one of the CfRadial experts?

Reply:
Thanks for this info! Also, what is the default `_FillValue` for the netcdf?

Reply:
This is where I took it from!
<img width="584" alt="Screenshot 2023-10-02 at 12 14 27 PM" src="https://github.com/openradar/xradar/assets/35923822/be61dd31-5508-4da2-b091-d8008dafa48a">

Reply:
You can get it from `netCDF4.default_fillvals` (netcdf4-python). Or you can grab it from `netcdf.h` in the netcdf-c sources ;-)

Reply:
> You can get it from `netCDF4.default_fillvals` (netcdf4-python). Or you can grab it from `netcdf.h` in the netcdf-c sources ;-)

Great, thank you! I'll take a look right away.

Reply:
If you perform a GitHub search for queries like `pyart "-32768"` or `lrose "-32768"`, you'll notice that in the code sections of various radar tools such as Pyart, Lrose Radx, Singledop, Multidop, Meteoswiss, and more, the value "-32768" is commonly utilized. However, as @mgrover1 said, we can use `netcdf default` as well.